File: createJson/src/main/java/com/ruzhan/jsonfile/model/HttpResult.java
package com.ruzhan.jsonfile.model;

/**
 * Created by ruzhan123 on 2018/7/3.
 */
public class HttpResult<T> {

    public int code;
    public String message;
    public T data;

    public HttpResult(int code, String message, T data) {
        this.code = code;
        this.message = message;
        this.data = data;
    }
}

File: common/src/main/java/com/ruzhan/lion/model/HttpResult.kt
package com.ruzhan.lion.model

import com.google.gson.annotations.SerializedName

/**
 * Created by ruzhan123 on 2018/7/4.
 */
data class HttpResult<T>(
    @SerializedName("code")
    var code: Int = 0,
    @SerializedName("message")
    var message: String? = null,
    @SerializedName("data")
    var data: T? = null
)
